tooling/bundler/src/bundle/windows/nsis.rs

```diff
@@ -68,6 +67,7 @@ pub fn bundle_project(settings: &Settings, updater: bool) -> crate::Result<Vec<P
   let nsis_toolset_path = tauri_tools_path.join("NSIS");
 
   if !nsis_toolset_path.exists() {
+    create_dir_all(&nsis_toolset_path)?;
```

Comment:
Creating an NSIS folder in advance will cause subsequent renaming failures after decompression

```rust
rename(_tauri_tools_path.join("nsis-3.08"), nsis_toolset_path)?;
```